src/jaegertracing/testutils/TracerUtil.h

```diff
@@ -41,6 +41,7 @@ struct ResourceHandle {
 };
 
 std::shared_ptr<ResourceHandle> installGlobalTracer();
+std::shared_ptr<ResourceHandle> installGlobalTracer(bool traceId128Bit);

```

Comment:
`installGlobalTracer(true)` at call site is very hard to read, I would rather call this explicitly `installGlobalTracer128Bit`

Reply:
done